src/main/target/OMNIBUSF7/target.mk

```diff
@@ -1,12 +1,13 @@
 F7X5XG_TARGETS += $(TARGET)
-FEATURES       += SDCARD VCP
+FEATURES       += SDCARD VCP ONBOARDFLASH
```

Comment:
I think this should get an if/else similar to the {REVO,FURYF3,FURYF4}/target.mk

Reply:
Good call mate. done